packages/react-components/react-tag-picker/library/src/components/TagPickerControl/useTagPickerControl.tsx

```diff
@@ -74,7 +77,17 @@ export const useTagPickerControl_unstable = (
   }
 
   const observerRef = useResizeObserverRef<HTMLSpanElement>(([entry]) => {
-    innerRef.current?.style.setProperty(tagPickerControlAsideWidthToken, `${entry.contentRect.width}px`);
+    const targetWindow = targetDocument?.defaultView;
+
+    if (targetWindow) {
+      if (rafIdRef.current) {
+        targetWindow.cancelAnimationFrame(rafIdRef.current);
+      }
+
+      targetWindow.requestAnimationFrame(() => {
```

Comment:
should `rafIdRef.current`  be assigned here?

Reply:
discussed in offline. no. `cancel` in callback also removed